x/swaprouter/export_test.go

```diff
@@ -15,6 +15,10 @@ func (k Keeper) GetOsmoRoutedMultihopTotalSwapFee(ctx sdk.Context, route types.M
 	return k.getOsmoRoutedMultihopTotalSwapFee(ctx, route)
 }
 
+func (k *Keeper) FundCommunityPool(ctx sdk.Context, fee sdk.Coins, sender sdk.AccAddress) error {
+	return k.communityPoolKeeper.FundCommunityPool(ctx, fee, sender)
+}
```

Comment:
Is this export needed? (Don't we use the method directly from the Distr Keeper any ways?)

Reply:
Yes. It is redundant. I might need to  delete it in the later commit.